components/sections/home-page/Hero.tsx

```diff
@@ -1,47 +1,71 @@
-      <main className="flex flex-col largeTablet:flex-row ">
-        <div className="flex flex-col items-start flex-1">
-          <Typography variant="preHeading" alignLarge="left">
-            {data?.title}
-          </Typography>
+    <SectionWrapper pt={115} pts={60} pbs={0}>
+      <ContainerWithLine>
+        <main>
+          <div className="flex largeTablet:items-center h-full flex-col gap-x-6 gap-y-10 largeTablet:flex-row mx-6">
+            <div className="flex-1 ">
+              <Typography alignSmall="left" alignLarge="left" variant="preHeading">
+                YOU GOT THIS
+              </Typography>
 
-          <div className="largeTablet:max-w-lg">
-            <Heading alignLarge="left">{data?.heading}</Heading>
-          </div>
+              <div className="relative">
+                <img className="absolute -left-[49px] largeTablet:-left-[68px] -top-[50px]" src="/icons/journey_ball.svg" alt="Starting point"/>
 
-          <div className="mt-8 mb-10 tablet:mt-4 ">
-            <p className="text-xl text-textPrimary opacity-70 largeTablet:leading-8">
-              {data?.description}
-            </p>
-          </div>
-          <div className="flex gap-4 items-center flex-col largeTablet:flex-row">
-            {data?.cta?.map((item, i) => (
-              <Button key={i} href={item.ctaLink} borderVariant={ i !== 0 ? "gray" : "neon" }>
-                {item.ctaLabel}
-              </Button>
-            ))}
-          </div>
-        </div>
-        <div className="flex mt-16 justify-center items-center flex-1 largeTablet:justify-end largeTablet:mt-0">
-          <div className="w-full h-full relative pointer-events-none max-w-[463px] largeTablet:max-h-[299px]">
-            <img alt="OpenSauced hero image" src={data?.image as unknown as string} />
+                <Heading component="h1" alignSmall="left" alignLarge="left">
+                  Your next open source journey $orange-to-yellowstarts here$orange-to-yellow
+                </Heading>
+              </div>
+              
+              <div className="py-10">
+                <Typography alignLarge="left" alignSmall="left" variant="body3">
+                  Start the path to your next contribution and join the global community of open source developers making an impact around the world.
+                </Typography>
+              </div>
+
+              <div className="flex gap-4 flex-col largeTablet:flex-row">
+              {data?.cta?.map((item, i) => (
+                <Button key={i} href={item.ctaLink} borderVariant={ i !== 0 ? "gray" : "neon" }>
+                  {item.ctaLabel}
+                </Button>
+              ))}
+              </div>
+              
+              <div className="pt-6 pb-10">
+                <Typography alignLarge="left" alignSmall="left" variant="body3">
+                  Working with team?
+                  <span>
+                    <Link href="/enterprise">
```

Comment:
```suggestion
                    <Link href="/enterprise">
```

This link is broken. And can this be `/teams` instead?

Reply:
We do not have a page on '/teams' also.